modules/oadp-restore-request-limit.adoc

```diff
@@ -0,0 +1,58 @@
+// Module included in the following assemblies:
+//
+// * backup_and_restore/application_backup_and_restore/troubleshooting.adoc
+
+:_content-type: CONCEPT
+[id="oadp-restore-request-limit_{context}"]
+= Pods crash or restart due to lack of memory or CPU
+
+You can set specific resource requests for memory or CPU if a Velero or Restic pod crashes due to a lack of either of them.
```

Comment:
@RichardHoch
I suggest you make the end of the sentence less ambiguous: "... due to a lack of either of those resources."

Reply:
Good idea; thanks.